codegen/core/src/main/scala/com/lightbend/akkasls/codegen/SourceGeneratorUtils.scala

```diff
@@ -113,7 +113,12 @@ object SourceGeneratorUtils {
     override def toString: String = {
       val suffix = if (semi) ";" else ""
       imports
-        .map(pkg => s"import $pkg$suffix")
+        .map { imported =>
+          if (imported == "com.google.protobuf.any.Any") {
+            s"import com.google.protobuf.any.{ Any => ScalaPbAny }$suffix"
```

Comment:
👍

Reply:
Isn't this used by java codegen as well?

Reply:
I guess that would have failed some test.

Reply:
In Java it's `com.google.protobuf.Any` (note the difference in `any.Any` at the end.

and in Java `Any` is perfectly fine.